CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+  * Customized apiserver subnet must be empty when migrating a cluster to enable [apiserver-vnet-integration](https://learn.microsoft.com/azure/aks/api-server-vnet-integration). If the subnet has resources in it, the migration won't be allowed.
+   
+* Bug Fixes:
+  * Metrics server v0.6.3 will be used to prevent frequent OOMKills on Metric server.
+  * Allowing zonal NodeClaims to facilitate NodeClaims and node creation on [Node Auto Provisioning](https://learn.microsoft.com/azure/aks/node-autoprovision) for workloads with zone affinity constraints.
+  * SSHAccess of nodepool will not be reset from Disabled to LocalUser by partial put. Put operation will be blocked with 400 error code.
+  * Fixed bug where the eTag property in 2024-02-02 preview, 2024-03-02 preview, and 2024-04-02 preview APIs was returned with the wrong case (returned etag, should have been eTag).
+  * Patch version 1.28.9 fixes [Bug - OpenAPI handler fails on duplicated path](https://github.com/kubernetes/kubernetes/issues/122668). 
+
+* Component Updates: 
+  * Istio-based service mesh add-on revision asm-1-19 has been upgraded to patch v1.19.10, asm-1-20 has been upgraded to patch v.1.20.6, and asm-1-21 has been upgraded to patch v1.21.2. Users can restart the workload pods to trigger re-injection of the newer patch version of istio-proxy. More information can be found [here](https://learn.microsoft.com/azure/aks/istio-upgrade).
+  * Linux and Windows addon-token-adapter image is updated to mcr.microsoft.com/aks/msi/addon-token-adapter:master.240510.2
+  * Preview API 2021-11-01 preview will be removed.
+  * Managed Prometheus image version updated to [images:6.8.12-main-05-21-2024](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md).
+  * Update Policy addon to v1.4.
+  * Update cloud node manager to v1.30.0 on AKS 1.30+, v1.29.4 on AKS 1.29+, v1.28.9 on AKS 1.28+, v1.27.17 on AKS 1.27+.
+  * Update [aks app routing operator image](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#023---2024-04-22) to v0.2.3.
+  * Upgrade Azure File CSI driver to v1.28.10  on AKS 1.27, v1.29.5  on AKS 1.28, v1.30.2  on AKS 1.29.
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202405.20.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202405.20.0.txt).
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202405.20.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202405.20.0.txt).
+  * Azure Linux image has been updated to [AzureLinux-202405.20.0](vhd-notes/AzureLinux/202405.20.0.txt).
+  * AKS Windows Server 2019 image has been updated to [AKSWindows-2019-17763.5820.240516](vhd-notes/AKSWindows/2019/17763.5820.240516.txt).
+  * AKS Windows Server 2022 image has been updated to [AKSWindows-2022-20348.2461.240516](vhd-notes/AKSWindows/2022/20348.2461.240516.txt).
+
```

Comment:
We can also add at the bottom - for up to date information on the latest images avaiable across platforms please go to https://releases.aks.azure.com/webpage/index.html

Reply:
Does the release tracker get updated as soon as this release goes live with the latest versions? 

If not it would be nice for maybe the release tracker to show the upcoming releases or keep the new versions here as the information is helpful.